api/classes/publisher.js

```diff
@@ -201,6 +201,18 @@ class BasePublisher {
     });
   }
 
+  updateProjectReadOnlyProperty(readonly) {
+    if (typeof readonly !== `boolean`) {
```

Comment:
when the request comes in, because we have Joi validation, it automatically converts it to a boolean value so this would never be a string (it can only be `null` or `undefined`).